pkg/k8shandler/visualization.go

```diff
@@ -488,3 +507,14 @@ func updateCurrentImages(current *apps.Deployment, desired *apps.Deployment) (*a
 
 	return current
 }
+
+func createSharedConfig(logging *logging.ClusterLogging, kibanaAppHost, kibanaInfraHost string) *v1.ConfigMap {
+	return utils.ConfigMap(
+		"sharing-config",
+		logging.Namespace,
+		map[string]string{
+			"kibanaAppHost":   kibanaAppHost,
```

Comment:
Is this a host or a url?  The arguments to `createSharedConfig` are urls

Reply:
updated name to be more appropriate